src/main/java/com/maxmind/geoip2/record/City.java

```diff
@@ -23,4 +33,8 @@ public Integer getConfidence() {
         return this.confidence;
     }
 
+    public static City empty() {
```

Comment:
Was there a particular reason you want with static factory methods for the empty objects instead of a constructor with no arguments?

Also, please add some basic documentation for these.